Bottom: 4281bcc6a82f5f962610fdb19afca40a721cbb51
Top:    526fe8e7b5cbac58e196a9a0926c852e114c1bd6
Author: Ian Kent <raven@themaw.net>
Date:   2013-02-27 13:17:03 +0800

Refresh of autofs-5.0.7-fix-libtirpc-build-option.patch

---

diff --git a/autofs.spec b/autofs.spec
index b8a3b7a..01f715c 100644
--- a/autofs.spec
+++ b/autofs.spec
@@ -12,6 +12,10 @@
 # disable them.
 %define with_systemd        %{?_without_systemd:        0} %{?!_without_systemd:        1}
 
+# Use --without libtirpc in your rpmbuild command or force values to 0 to
+# disable them.
+%define with_libtirpc        %{?_without_libtirpc:        0} %{?!_without_libtirpc:        1}
+
 Summary: A tool from automatically mounting and umounting filesystems.
 Name: autofs
 %define version 5.0.7
@@ -25,6 +29,9 @@ Buildroot: %{_tmppath}/%{name}-tmp
 %if %{with_systemd}
 BuildRequires: systemd-units
 %endif
+%if %{with_libtirpc}
+BuildRequires: libtirpc-devel
+%endif
 BuildPrereq: autoconf, hesiod-devel, openldap-devel, bison, flex, cyrus-sasl-devel
 Prereq: chkconfig
 Requires: /bin/bash mktemp sed textutils sh-utils grep /bin/ps
